mphys/mphys_group.py

```diff
@@ -31,12 +31,14 @@ def configure(self):
     def _mphys_promote_by_tag(self, iotype, tag):
         for subsystem in self.mphys_subsystems:
             promoted = []
+            print(f'promoting from {subsystem.name}')
```

Comment:
Is this for debugging? I suggest we either remove it or add an if statement to print the information from the root proc only.

Reply:
I have removed this